include/inexor/vulkan-renderer/world/collision.hpp

```diff
@@ -0,0 +1,250 @@
+#pragma once
```

Comment:
This header is very costly to include. Maybe you could add some explicit instantiations?

Reply:
- [ ] Add explicit instantiations of `RayCubeCollision<Cube>` in `collision.cpp`.